src/main.cpp

```diff
@@ -656,6 +673,26 @@ int main(int argc, char *argv[])
                 params.codec = optarg;
                 break;
 
+	    case 'r':
+	        params.framerate = atoi(optarg);
+	 	break;
+
+            case 'x':
```

Comment:
Also indentation white space mismatch.

Reply:
Hmm... I guess I need to figure out a way to tell Emacs to not use tabs at all. It is still doing it even when I try to set the coding style from existing code. Thank you for the notification; I couldn't see anything was off at all. Emacs has certainly given me a really big hassle contributing to this project with its strange automatic indents.